Dockerfile

```diff
@@ -49,13 +51,13 @@ EXPOSE 8000
 ENV DJANGO_SETTINGS_MODULE=my_project.settings.deploy
 
 # Call collectstatic (customize the following line with the minimal environment variables needed for manage.py to run):
-RUN DATABASE_URL='' /venv/bin/python manage.py collectstatic --noinput
+RUN DATABASE_URL='' python manage.py collectstatic --noinput
 
 # Tell uWSGI where to find your wsgi file (change this):
 ENV UWSGI_WSGI_FILE=my_project/wsgi.py
 
 # Base uWSGI configuration (you shouldn't need to change these):
-ENV UWSGI_VIRTUALENV=/venv UWSGI_HTTP=:8000 UWSGI_MASTER=1 UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_UID=1000 UWSGI_GID=2000 UWSGI_LAZY_APPS=1 UWSGI_WSGI_ENV_BEHAVIOR=holy
+ENV UWSGI_HTTP=:8000 UWSGI_MASTER=1 UWSGI_HTTP_AUTO_CHUNKED=1 UWSGI_HTTP_KEEPALIVE=1 UWSGI_LAZY_APPS=1 UWSGI_WSGI_ENV_BEHAVIOR=holy
```

Comment:
It might be more readable to put these in a uwsgi.ini file.

Reply:
True, but I like that they can be changed via the environment, as noted in the post. Plus, it's one less file to worry about.

Reply:
If we were to switch to a config file I'd be tempted to switch to gunicorn at the same time, given our greater familiarity with it and the fact that it seems to crash with obscure errors less often...